PizzaOvenUI/Screen_AwaitStart.qml

```diff
@@ -30,10 +30,17 @@ Item {
         theCircle.animate();
         circleContent.animate();
         screenEntryAnimation.start();
-        rootWindow.displayedDomeTemp = menuSettings.json.menuItems[foodIndex].domeTemp;
-        rootWindow.displayedStoneTemp = menuSettings.json.menuItems[foodIndex].stoneTemp;
-        localDomeTemp = menuSettings.json.menuItems[foodIndex].domeTemp;
-        localStoneTemp = menuSettings.json.menuItems[foodIndex].stoneTemp;
+        rootWindow.displayedDomeTemp = upperFront.setTemp;
+        rootWindow.displayedStoneTemp = lowerFront.setTemp;
+        localDomeTemp = upperFront.setTemp;
+        localStoneTemp = lowerFront.setTemp;
```

Comment:
We don't want to load the temps from the menuSettings file because if the user customized the settings (and they are different than the custom cycle), loading from the file would override the user settings with the values from the file. This is the original bug we were going after here.